src/com/lilithsthrone/game/character/body/types/EyeType.java

```diff
@@ -4,6 +4,9 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.function.Function;
+
+import org.h2.util.StringUtils;
```

Comment:
This package isn't part of the the standard JDK and thus we can't use it unless the dependency is added.

Reply:
Oops, that wasn't the one I wanted anyway.

I did notice the project has no dependencies. Is that a deliberate choice? I can certainly rework this without bringing in a dependency.

Besides my botched import, any thoughts?